File: Assignments/Assignment_16/p7.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        int size=(row*2)-1,num=1;
        for(int i=0;i<row;i++){

                for(int j=1;j<=size;j++){

                        if((j<(i+1))||(j>(size-i))){

                                printf("\t");
                        }else{

                                printf("%d\t",num);
                                num++;
                        }
                }printf("\n\n");
        }

}

File: JAVA/8.Nested_For_1/p7.java
import java.util.*;

class Demo{

	public static void main(String[] args){
	
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter no. of rows: ");
		int row = sc.nextInt();
		for(int i=row;i>0;i--){
		
			for(int j=row;j>0;j--){
			
				System.out.print(j +"\t");
			}System.out.println();
		}
	}
}
